free-1.md

```diff
@@ -57,12 +153,32 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Probably half and half. If a nice language design can be done to meet some/all
+of these goals, then it would be a useful DSL. Since there are very few examples
+of DSLs in this domain that seem to be have "a nice language design", a lot of
+effort will have to be spent on making design decisions.
+
+Since this DSL will probably have to parse code that the user inputs, a lot of
+effort will have to be spent on that front as well. Though, with a good language
+design, the parsing and background abstraction translating/compiling should be
+relatively easier.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Seeing as there aren't that many DSLs in this domain, and depending on how much
+of the web dev things this DSL should abstract, this can be quite a large idea.
```

Comment:
Yeah, it'll probably be important to get a solid idea of how much abstraction you want before beginning work on the language, both to make implementing it easier and to avoid feature creep.